packages/kolme-store/src/error.rs

```diff
@@ -1,15 +1,38 @@
 use merkle_map::{MerkleSerialError, Sha256Hash};
 
-#[derive(thiserror::Error, Debug)]
+#[derive(Debug, Clone, Copy, serde::Serialize, serde::Deserialize)]
+pub enum StorageBackend {
+    Fjall,
+    Postgres,
+    InMemory,
+    Unknown,
```

Comment:
Do we actually need this? We should have a fully defined set of available storage backends.